funcx_sdk/funcx/sdk/web_client.py

```diff
@@ -187,6 +187,9 @@ def register_endpoint(
             data.update(additional_fields)
         return self.post("/endpoints", data=data)
 
+    def get_result_amqp_url(self) -> globus_sdk.GlobusHTTPResponse:
```

Comment:
I didn't look at the services PR a few weeks ago, but I would have suggested 'amqp_result_connection_url' or something without the 'get', being redundant with requests.GET.  But too late now.